ci-operator/step-registry/baremetal/lab/post/firewall/baremetal-lab-post-firewall-commands.sh

```
@@ -16,11 +16,16 @@ SSHOPTS=(-o 'ConnectTimeout=5'
     test -f /var/builds/"${NAMESPACE}"/preserve && \
   exit 0
 
+CLUSTER_NAME=$(<"${SHARED_DIR}/cluster_name")
+
 if [ x"${DISCONNECTED}" != x"true" ]; then
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)